oak_functions/examples/weather_lookup/module/src/tests.rs

```diff
@@ -115,21 +114,25 @@ async fn test_server() {
     mock_static_server_background.terminate_and_join().await;
 }
 
-async fn make_request(port: u16, request_body: &[u8]) -> Vec<u8> {
-    let client = Client::builder().http2_only(true).build_http();
-    let request = hyper::Request::builder()
-        .method(http::Method::POST)
-        .uri(format!("http://localhost:{}/invoke", port))
-        .body(hyper::Body::from(request_body.to_vec()))
-        .unwrap();
+async fn make_request(port: u16, request_body: &[u8]) -> Response {
```

Comment:
Should this go to the utils package? I think it will be used identically in all examples.